downstream/modules/eda/proc-eda-create-event-stream.adoc

```diff
@@ -0,0 +1,48 @@
+[id="eda-create-event-stream"] 
+
+= Creating an event stream
+
+You can create event streams that will be attached to a rulebook activation. 
+
+.Prerequisites
+
+* If you will be attaching your event stream to a rulebook activation, ensure that your activation has a decision environment and project already set up.
+* If you plan to connect to {ControllerName} to run your rulebook activation, ensure that you have created a {PlatformName} credential type in addition to the decision environment and project. For more information, see xref:eda-set-up-rhaap-credential[Setting up a {PlatformName} credential].
+
+.Procedure
+
+. Log in to {PlatformNameShort}.
+. From the navigation panel, select {MenuADEventStreams}.
+. Click btn:[Create event stream].
+. Insert the following:
++
+Name:: Insert the name.
+Organization:: Click the list to select an organization or select *Default*.
+Event stream type:: Select the event stream type you prefer.
++
+[NOTE]
+====
+This list displays at least 10 default event stream types that can be used to authenticate the connection coming from your remote server.
+====
+Credentials:: Select a credential from the list, preferably the one you created for your event stream.
+Headers:: Enter HTTP header keys, separated by commas, that you want to include in the event payload. To include all headers, leave the field empty.
+
+Forward events to rulebook activation:: Use this option to enable or disable the capability of forwarding events to rulebook activations.
++
+[NOTE]
+====
+The event stream's event forwarding can be disabled for testing purposes while diagnosing connections and evaluating the incoming data. Disabling the *Forward events to rulebook activation* option allows you to test the event stream connection with the remote system, analyze the header and payload, and if necessary, diagnose credential issues. This ensures that events will not be forwarded to rulebook activations causing rules and conditions to be triggered inadvertently while you are in test mode. Some enterprises might have policies to change secrets and passwords at regular cadence. You can enable/disable this option anytime after the event stream is created.
+====
+
+. Click btn:[Create event stream].
+
+After creating your event stream, the following outputs occur:
+
+* The Details page is displayed. From there or the Event Streams list view, you can edit or delete it. Also, the Event Streams page shows all of the event streams you have created and the following columns for each event: *Events received*, *Last event received*, and *Event stream type*. As the first two columns receive external data through the event stream, they will continuously update to let you know they are receiving events from remote systems.
```

Comment:
As the first two columns receive external data through the event stream, they continuously update with event information received from remote systems. ?

Reply:
I'm choosing to go with "As the first two columns receive external data through the event stream, they **are** continuously **updated** to let you know they are receiving events from remote systems."